CHANGELOG.md

```diff
@@ -165,60 +63,86 @@ testing and greater compatibility with dependent projects.
 - `[rpc]` The subscription client interface provides a structured `Query`
   mechanism to help ensure compile-time validity of subscription queries.
   See the crate docs and [#584] for details.
 - `[rpc]` The RPC request and response types' fields are now all publicly
   accessible ([#636]).
-
-- `[tendermint | rpc | light-client]` Crates now compile to WASM on the `wasm32-unknown-unknown` and `wasm32-wasi` targets ([#463])
-- Dependency updates: 
-    - Update sled to 0.34 ([#490])
-    - Update k256 to v0.5 ([#578])
-    - Remove tai64 crate  ([#603])
+- `[rpc]` A new RPC probe (in the `rpc-probe` directory) has been added to
+  facilitate quick, pre-scripted interactions with a Tendermint node (via its
+  WebSocket endpoint). This aims to help improve testing and compatibility
+  between Tendermint in Go and Rust. ([#653])
+- `[rpc]` The `WebSocketClient` now adds support for all remaining RPC requests
+  by way of implementing the `Client` trait ([#646])
+- `[rpc]` Support for the `tx_search` RPC endpoint has been added ([#701])
+- `[rpc]` Responses that include events now automatically have their tag
+  key/value pairs decoded from base64, where previously tag key/value pairs
+  were Base64-encoded ([#717])
+- `[rpc]` Support for the `consensus_state` RPC endpoint has been added ([#719])
+- `[tendermint]` Bech32 encoding fix ([#690])
+- `[tendermint]` Remove `total_voting_power` parameter from `validator::Set::new` ([#739])
+- `[tendermint, rpc, light-client]` Crates now compile to WASM on the
+  `wasm32-unknown-unknown` and `wasm32-wasi` targets ([#463])
+- `[tendermint, light-client]` Specify the proposer in the validator set of fetched light blocks ([#705])
+- `[tendermint-proto]` Upgrade protobuf definitions to Tendermint Go v0.34.0 ([#737])
+- `[testgen]` Compute `last_block_id` hash when generating a `LightChain` ([#745])
+- Dependency updates:
+  - Update sled to 0.34 ([#490])
+  - Update k256 to v0.7 ([#752])
+  - Remove tai64 crate  ([#603])
 
 ### BUG FIXES:
 
+- `[light-client]` Fix bug where a commit with only absent signatures would be
+  deemed valid instead of invalid ([#650])
+- `[light-client]` Revert a change introduced in [#652] that would enable DoS attacks,
+  where full nodes could spam the light client with massive commits (eg. 10k validators).
 - `[rpc]` Correctly handles control and keep-alive messages ([#516], [#590])
 - `[rpc]` More robust handling of concurrency issues ([#311], [#313])
+- `[tendermint]` CanonicalBlockId is now correctly decoded to `None`
+  in `CanonicalVote` and `CanonicalProposal` when its hash is empty.
+- `[tendermint]` Time serialization fix (part of [#665])
+- `[tendermint]` Fix abci::Data serialization to base64-encoded string. ([#667])
+- `[tendermint]` Simplify abci::Log serialization ([#667])
+- `[tendermint]` consensus::State backwards compatibility for deserialization ([#679])
+- `[tendermint-proto]` Timestamp serialization fix (part of [#665])
```

Comment:
I guess these might have also been opened because of the upgrade, although there might be fixes in there. I'm not sure.

Reply:
We could easily take them out if you want?

```suggestion
```